### PR TITLE
sudoku game with 3 different levels

### PR DESCRIPTION
**Pull request by ssachis**

### CODE REVIEW

File: sudoku/styles.css [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+body.dark {
+  background-color: rgb(45, 100, 117);
+  color: white;
+}
+header {
+  text-align: center;
+  padding-bottom: 20px;
+  padding-top: 10px;
+  border-bottom: 1px solid green;
+}
```
Comment (Abirpal202049) [inline sudoku/styles.css:13]: Dont target the element directly target it by class or id

**Review by Abirpal202049 — CHANGES_REQUESTED** [review]

File: sudoku/styles.css (hunk 1 above)
Comment (Abirpal202049) [reply sudoku/styles.css:13]: Target all the element by Id or Class don't target it directly

**Review by nandini7637 — CHANGES_REQUESTED** [review]
Kindly follow the folder structure. This is game no 1

File: sudoku/styles.css (hunk 1 above)
Comment (thepranaygupta) [reply sudoku/styles.css:13]: also format the files properly

**Comment by ssachis** [comment]
> Kindly follow the folder structure. This is game no 1

should I rename the folder to games/game_01
also after the changes should I make a new pr?

**Comment by nandini7637** [comment]
> > Kindly follow the folder structure. This is game no 1
> 
> should I rename the folder to games/game_01

Yeah and add code in assets folder like other folders.

**Review by Abirpal202049 — CHANGES_REQUESTED** [review]
In CSS Don't target the element directly target by using class or id

**Pull request closed without merging**